examples/quickstart-mlx/README.md

```diff
@@ -129,30 +107,30 @@ data = (
     test_partition["label"].astype(np.uint32),
 )
 
-train_images, train_labels, test_images, test_labels = map(mlx.core.array, data)
+train_images, train_labels, test_images, test_labels = map(mx.array, data)
 ```
 
 ### The model
```

Comment:
@charlesbvll how about if we remove the model block. in the end this is not so Flower-specific...

Reply:
I think we could remove the code (so we don't have to maintain the same code in 2 different places), but it might be good to keep some explanation of what the model is and what it is doing to give some context